### PR TITLE
CSS polish for cursors, hover, and misc.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Fixes https://github.com/flutter/devtools/issues/865, 2) from https://github.com/flutter/devtools/issues/705, and some misc cleanup.

### CODE REVIEW

File: packages/devtools/lib/src/memory/memory_chart.dart [hunk 1]
```diff
@@ -18,8 +18,7 @@ class MemoryChart extends CoreElement {
     element.id = _memoryGraph;
     element.style
       ..boxSizing = 'content-box' // border-box causes right/left border cut.
-      ..height = '${FramesBarChart.chartHeight}px'
-      ..width = '100%';
```
Comment (terrylucas) [inline packages/devtools/lib/src/memory/memory_chart.dart]: Quick question was 100% causing a problem or just unnecessary?

Comment (kenzieschmoll) [reply packages/devtools/lib/src/memory/memory_chart.dart]: width = '100%' was causing the right border of the graph to be cut off.  The chart still takes up the full width without this line, and the border is now visible 👍

Comment (terrylucas) [reply packages/devtools/lib/src/memory/memory_chart.dart]: Great! Thx.

File: packages/devtools/web/styles.css [hunk 2]
```diff
@@ -765,6 +774,14 @@ div.tabnav-tab {
     height: 30px;
 }
 
+.form-select:hover {
+    background-color: var(--button-selected-color);
+}
+
+input {
+    cursor: pointer;
```
Comment (jacob314) [inline packages/devtools/web/styles.css]: A pointer as the cursor for input seems surprising. For example, I expect this isn't right for code mirror, etc.
Can we use some more specific css rules to use a pointer cursor where needed?

Comment (kenzieschmoll) [reply packages/devtools/web/styles.css]: good point. I did this to target checkboxes - i'll change to `input[type='checkbox']`

**Pull request merged**